universal-application-tool-0.0.1/conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,31 @@
+# --- JSON-based tables
+
+# --- !Ups
+create table if not exists applicants (
+  id bigserial primary key,
+  object jsonb not null
+  -- No other constraint on applicant - empty object acceptable.
+);
+
+create table if not exists questions (
+  id bigserial primary key,
+  object jsonb not null,
+  check (object ? 'target')
```

Comment:
Question: is `'target'` here a jq path to where the question's data is stored?

Reply:
Yep!